relm4/src/factory/sync/collections/vec_deque.rs

```diff
@@ -579,3 +579,27 @@ impl<C: FactoryComponent> FactoryVecDeque<C> {
         output
     }
 }
+
+/// Adds a get_init function to FactoryComponents that are clonable
+trait CloneableFactoryComponent: FactoryComponent {
+    fn get_init(&self) -> Self::Init;
+}
```

Comment:
Looks good overall, but this trait must be public, otherwise nobody will be able to implement it. I suggest we move it to `relm4/src/factory/sync/traits.rs` where it can be located much easier.